src/lib/stylesheet.js

```diff
@@ -29,6 +27,11 @@ export default class StyleSheet {
     this._tags = []
     this._injected = false
     this._rulesCount = 0
+
+    const node =
+      typeof document === 'object' &&
```

Comment:
Here you can use `this._isBrowser`

Reply:
I feel like changing this might break [my test](https://github.com/zeit/styled-jsx/pull/482/files#diff-be6d9079ddafc0af10f17b86a4033dcaR109) :worried: because there isn't any browser testing implemented in this repo :frowning_face:

Reply:
probably in your test you can do `global.window = true`.

Reply:
ah no you can pass `isBrowser: true` to `makeRegistry` https://github.com/zeit/styled-jsx/blob/master/test/stylesheet-registry.js#L8-L12

Reply:
@giuseppeg Will this not cause unintended side effects because we are telling the application it is running in a browser and it's not really? If somewhere else in the application it checks `isBrowser` and if I set it to true, it might try to execute some other type of browser only script.

Reply:
this is an isolated unit test so it is perfectly fine. If you are talking about  `src/lib/stylesheet.js` instead that's fine too since you are checking that `typeof document === 'object'` is defined i.e. you are in a browser environment.

Reply:
I was right about the unintended side effects. It's possible that `isBrowser == true` and `typeof document === 'undefined'`, which will create an error.

Reply:
it should behave the same when isBrowser is true. If there are failing tests this is a good thing and we should look into this issue. In a real browser `typeof document === 'object'` will be the same as using `this._isBrowser` because the latter is just `typeof window !== 'undefined;`

Reply:
If you want me to break the tests, I will, I just don't have time to fix them myself.